backend/src/main/java/com/redhat/cloud/notifications/recipients/itservice/ITUserServiceWrapper.java

```diff
@@ -0,0 +1,22 @@
+package com.redhat.cloud.notifications.recipients.itservice;
+
+import com.redhat.cloud.notifications.recipients.itservice.pojo.request.ITUserRequest;
+import com.redhat.cloud.notifications.recipients.itservice.pojo.response.ITUserResponse;
+import io.smallrye.mutiny.Uni;
+import org.eclipse.microprofile.rest.client.inject.RestClient;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import java.util.List;
+
+@ApplicationScoped
+public class ITUserServiceWrapper {
+
+    @Inject
+    @RestClient
+    ITUserService itUserService;
+
+    public Uni<List<ITUserResponse>> getUsers(String accountId, boolean adminsOnly) {
```

Comment:
Not to apply right now - but we might want to optimize this query - Would it be possible in the future to also pass the list of users that we want to fetch? 

In most cases (but not all cases) we have the list of users (i.e. all the subscribed) that we would like to filter for.

Reply:
Generally it's possible, yes. We can find users by a lot of attributes. If we need it, we can just add it.

Reply:
I can add a link to the gitlab IT User Service Readme if you want, @josejulio ?

Reply:
I think we need to store that somewhere for easier access, but not sure if on this README or somewhere else.

Reply:
It's already in the jira ticket. So in case we really need it and nobody knows the url, we are still able to find it via git blame